doc/requirements.txt

```diff
@@ -3,4 +3,4 @@ sphinx-rtd-theme>=1.0.0
 sphinxcontrib-bibtex>=2.4.2
 python_docs_theme
 furo
-omotes-simulator-core
\ No newline at end of file
+.
```

Comment:
why do you use the dot?

Reply:
To install the omotos-simulator-core

Reply:
This is needed to make autodoc work

Reply:
Ok it is the requirement file for the documentation. I missed that part of the change.